File: Organization Of Programming Languages/Organization Of Programming Languages/BubbleSort.java
import java.util.Arrays;
public class BubbleSort{
   public static void main(String[] args){
      int[] arr = new int[]{34, 56, 4, 10, 77, 51, 93, 30, 5, 52};
      System.out.println("Unsorted Array: " + Arrays.toString(arr));

	  // Bubble Sort		
	  for(int i = 0; i < arr.length-1; i++)
     {
	      for (int j = i+1; j < arr.length; j++)
         {
            //If greater, swap ints
				if (arr[i] > arr[j])
				{
					int temp = arr[i];
					arr[i] = arr[j];
					arr[j] = temp;
				}
			}
		}
      System.out.println("\nSorted Array: " + Arrays.toString(arr));
   }
}

File: Organization Of Programming Languages/BubbleSort.js
// Creating the bblSort function
 function bubbleSort(arr){
console.log("Unsorted array");
console.log(arr);
//First loop
 for(var i = 0; i < arr.length; i++){
   //Nested loop
   for(var j = 0; j < ( arr.length - i -1 ); j++){

     // If greater, swap
     if(arr[j] > arr[j+1]){
       var temp = arr[j]
       arr[j] = arr[j + 1]
       arr[j+1] = temp
     }
   }
 }
 // Print the sorted array
 console.log("Sorted array");
 console.log(arr);
}


// This is our unsorted array
var arr = [34, 56, 4, 10, 77, 51, 93, 30, 5, 52];


// Now pass this array to the bblSort() function
bubbleSort(arr);
